web/libs/editor/src/tags/object/RichText/view.jsx

```diff
@@ -362,7 +362,7 @@ class RichTextPieceView extends Component {
   render() {
     const { item } = this.props;
 
-    if (!item._value) return null;
+    if (!item._value === null) return null;
```

Comment:
this comparison doesn't make sense and will always return false because you are comparing boolean value with `null`. what are you trying to achieve here?

Reply:
my bad, the idea would be validate if `item._value` is null not `!item._value`. It's fixed

Reply:
if you are checking that value is missing (like it's `undefined` or `null`) then it's better to use `isDefined()`